elide-datastore/elide-datastore-hibernate5/src/main/java/com/yahoo/elide/datastores/hibernate5/HibernateStore.java

```diff
@@ -236,6 +244,92 @@ public void commit() {
         }
 
         @Override
+        public Collection filterCollection(Collection collection, String type, FilterScope<?> filterScope) {
```

Comment:
This code looks exactly the same as Hibernate 3 - anyway we can commonize the common parts?